cv-corpus-23.0-2025-09-17/draft/zh-TW/nan-tw.md

```diff
@@ -0,0 +1,199 @@
+# *台語* &mdash; Taiwanese (Minnan) (`nan-tw`)
```

Comment:
This datasheet appears to be a mix of `zh-TW` and `en` ?  Could you stick to one? Either is fine, if it's `en` it should go in the `en` folder. For the first version of the platform we will only be providing a single datasheet per language, but you can choose which language it is in.

Reply:
What I’m thinking is add a Chinese datasheet inside zh-tw/ and an English one in en/ the same time. 

For “single datasheet per language”, you means 
1) one cv language can only have one datasheet either in which language, or 
2) one cv language can have multiple datasheets in different language, but we need to only use one language inside that datasheet (that is, we need to translate the section title and default contents)

Reply:
that's right. A note about providing a datsheet in a new local (e.g. nan-tw): this is great, but just make sure everything is translated (e.g. the boiler plate content in the en template should be translated as well).